app/stories/shared.pcss

```diff
@@ -3,9 +3,6 @@
 }
 
 .svgIcon {
-  display: block;
-  max-height: 1em;
-  max-width: 1em;
```

Comment:
@mondoreale I set storybook to display icons at their "native" size, this could be why they look smaller in storybook

Reply:
i.e. 
![image](https://user-images.githubusercontent.com/43438/57594523-21301980-7573-11e9-9b5c-bd2ced16de6a.png)

Some icons need explicit sizing to look correct, and without this it's not clear which ones. Still not entirely ideal since this doesn't show if/how the icons will scale, but at least this shows what the icon *actually* looks like when not being explicitly sized. If we want them to be uniform by default then we should probably clean up the sizes in the svgs or apply default dimensions in the svg icon wrapper.